test/index.test.js

```diff
@@ -19,7 +19,7 @@ const HasManyDissociator = require('../src/services/has-many-dissociator');
     const models = {};
     const sequelizeOptions = {
       Sequelize,
-      connections: [sequelize],
+      connections: { sequelize },
```

Comment:
Totally my fault, did not catch that during the review. This would have save us from this whole PR ...